pkgs/applications/misc/hellwal/default.nix

```
@@ -0,0 +1,43 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  makeWrapper,
+}:
+
+let
+  pkgs = import <nixpkgs> {};
+in
+pkgs.stdenv.mkDerivation rec {
```

Prefer `finalAttrs` over `rec` (see https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes)
```suggestion
stdenv.mkDerivation (finalAttrs: {
```